templates/repo/commits_list.tmpl

```diff
@@ -1,87 +1,74 @@
-<div class="ui attached table segment commit-table">
-		<table class="ui very basic striped table unstackable fixed" id="commits-table">
-			<thead>
-				<tr>
-					<th class="three wide">{{.locale.Tr "repo.commits.author"}}</th>
-					<th class="two wide sha">SHA1</th>
-					<th class="nine wide message">{{.locale.Tr "repo.commits.message"}}</th>
-					<th class="two wide right aligned">{{.locale.Tr "repo.commits.date"}}</th>
-				</tr>
-			</thead>
-			<tbody class="commit-list">
-				{{$commitRepoLink := $.RepoLink}}{{if $.CommitRepoLink}}{{$commitRepoLink = $.CommitRepoLink}}{{end}}
-				{{range .Commits}}
-					<tr>
-						<td class="author">
-							{{$userName := .Author.Name}}
-							{{if .User}}
-								{{if .User.FullName}}
-									{{$userName = .User.FullName}}
-								{{end}}
-								{{avatar $.Context .User 28 "gt-mr-2"}}<a href="{{.User.HomeLink}}">{{$userName}}</a>
+<div class="ui attached table segment commits-list">
+	{{$commitRepoLink := $.RepoLink}}{{if $.CommitRepoLink}}{{$commitRepoLink = $.CommitRepoLink}}{{end}}
+	{{range .Commits}}
+		<div class="commits-list-row gt-df gt-ac gt-px-4 gt-py-2">
```

Comment:
These "gt-df gt-ac gt-px-4 gt-py-2" seems quite wordy.

Since it has a class `commits-list-row`, is it better to put the styles on the particular CSS class name?

Reply:
As mentioned in the other issue, I think what can be done with helper classes should be done with them. I see CSS currently only for stuff that can not be done with helper classes.

Another benefit of helper classes is that they move when the elemen moves, so won't leave garbage CSS during refactoring.

I do like to retain descriptive class names, primarily as a visual aid for reading the code.

Reply:
> I think what can be done with helper classes should be done with them.

Totally agree. It helps when working on other people's code.


> I do like to retain descriptive class names, primarily as a visual aid for reading the code.

Interesting idea. I'll give that a try,